backend/app/graphql/all_users.py

```diff
@@ -0,0 +1,39 @@
+from ..services.implementations.user_service import UserService
+import graphene
+import logging
+from .types import QueryList
+
+
+class UserType(graphene.ObjectType):
+    name = graphene.String()
+    email = graphene.String()
+    role = graphene.String()
+
+
+class AllUsersQuery(QueryList):
```

Comment:
Normally, this would be a `Query`, not a `QueryList`. My suggestion would be to put both the `all_users` and `user_by_id` queries in the same file, make them inherit from `Query`, and make them both children of a `QueryList`.